docs/src/writing_good_rules.md

```diff
@@ -247,23 +262,22 @@ In this sense they wrap either a natural or structural tangent.
 
 ## Use `@not_implemented` appropriately
 
-One can use [`@not_implemented`](@ref) to mark missing differentials.
-This is helpful if the function has multiple inputs or outputs, and one has worked out analytically and implemented some but not all differentials.
+You can use [`@not_implemented`](@ref) to mark missing differentials.
+This is helpful if the function has multiple inputs or outputs, and you have worked out analytically and implemented some but not all differentials.
 
 It is recommended to include a link to a GitHub issue about the missing differential in the debugging information:
 ```julia
 @not_implemented(
-"""
-derivatives of Bessel functions with respect to the order are not implemented:
-https://github.com/JuliaMath/SpecialFunctions.jl/issues/160
-"""
+    """
+    derivatives of Bessel functions with respect to the order are not implemented:
+    https://github.com/JuliaMath/SpecialFunctions.jl/issues/160
+    """
 )
 ```
 
 Do not use `@not_implemented` if the differential does not exist mathematically (use `NoTangent()` instead).
 
-While this is more verbose, it ensures that if an error is thrown during the `pullback` the [`gensym`](https://docs.julialang.org/en/v1/base/base/#Base.gensym) name of the local function will include the name you gave it.
-This makes it a lot simpler to debug from the stacktrace.
+Note: [ChainRulesTestUtils.jl](https://github.com/JuliaDiff/ChainRulesTestUtils.jl) marks `@not_implemented` differentials as "test broken".
```

Comment:
To document https://github.com/JuliaDiff/ChainRulesTestUtils.jl/pull/218

Reply:
It did that even before. for most cases.
The case it didn't handle was for things where the tangent was determined to be `NoTangent`.

But regardless this is a good change